es-utils/pom.xml

```diff
@@ -63,7 +63,7 @@
 				<plugin>
 					<groupId>org.jacoco</groupId>
 					<artifactId>jacoco-maven-plugin</artifactId>
-					<version>0.7.5.201505241946</version>
+					<version>0.8.5</version>
```

Comment:
Please create `property` and use that for version value.

Reply:
done